src/protos/io/defang/v1/fabric.proto

```diff
@@ -326,6 +331,29 @@ message ListConfigsRequest {
 
 message ListConfigsResponse { repeated ConfigKey configs = 1; }
 
+enum DeploymentState {
```

Comment:
Where is this used?

Reply:
Thanks for catching this. This should be removed.